drools/src/main/java/com/datatorrent/drools/operator/StatelessDroolsOperator.java

```diff
@@ -140,4 +164,124 @@ public StatelessKieSession getKieSession()
   {
     return kieSession;
   }
+
+  private class RulesFiredListener implements AgendaEventListener
```

Comment:
I tried both implementation of DefaultAgendaEventListener it is printing the event on console when we hit the event